src/Avalonia.Base/Utilities/TypeUtilities.cs

```diff
@@ -93,13 +93,25 @@ public static bool AcceptsNull(Type type)
             return !type.IsValueType || IsNullableType(type);
         }
 
+        /// <summary>
+        /// Returns a value indicating whether null can be casted to the specified type.
```

Comment:
Here `null` seems to be a typo. Maybe `value` or `object`?

Reply:
```suggestion
        /// Returns a value indicating whether value can be casted to the specified type.
```